rosdep/python.yaml

```
@@ -7078,6 +7079,7 @@ python3-lark-parser:
     xenial: [python3-lark-parser]
 python3-lttng:
   alpine: [py3-lttng]
+  arch: [python-lttng-ust]
```

isn't it `python-lttngust`?

```suggestion
  arch: [python-lttngust]
```

---

Good catch thanks!